src/app/components/ChartbeatAnalytics/utils/index.test.ts

```diff
@@ -794,483 +1024,56 @@ describe('Chartbeat utilities', () => {
-    );
-
-    expect(getConfig(fixtureData)).toStrictEqual(expectedConfig);
-  });
-
-  it('should return config for canonical pages when page type is IDX and env is not live', () => {
-    const fixtureData: GetConfigProps = {
-      isAmp: false,
-      platform: 'canonical',
-      pageType: INDEX_PAGE,
-      data: {},
-      brandName: 'BBC-Persian',
-      chartbeatDomain: 'bbc.co.uk',
-      env: 'test',
-      service: 'persian',
-      origin: 'test.bbc.com',
-      previousPath: '/previous-path',
-    };
-
-    const expectedConfig = {
-      domain: 'test.bbc.co.uk',
-      idSync: {
-        bbc_hid: 'foobar',
-      },
-      path: '/',
-      sections: 'Persian, Persian - IDX',
-      title: 'This is an index page title',
-      type: 'Index',
-      uid: 50924,
-      useCanonical: true,
-      virtualReferrer: 'test.bbc.com/previous-path',
-    };
-
-    const mockTitle = jest
-      .fn()
-      .mockImplementation(() => 'This is an index page title');
-
-    // @ts-expect-error requires mocking for testing purposes
-    frontPageUtils.getPageTitle = mockTitle;
-
-    const expectedCookieValue = 'foobar';
-    (jest.spyOn(Cookie, 'get') as jest.Mock).mockImplementation(
-      () => expectedCookieValue,
-    );
-
-    expect(getConfig(fixtureData)).toStrictEqual(expectedConfig);
-  });
-
-  it('should return null for virtualReferrer when there is no previousPath', () => {
-    // @ts-expect-error testing partial data to ensure behaviour is as expected
-    const fixtureData: GetConfigProps = {
-      isAmp: false,
-      platform: 'canonical',
-      pageType: FRONT_PAGE,
-      data: {},
-      brandName: 'BBC-News',
-      chartbeatDomain: 'bbc.co.uk',
-      env: 'test',
-      service: 'news',
-      origin: 'test.bbc.com',
-    };
-
-    const chartbeatConfig = getConfig(fixtureData);
-    expect(chartbeatConfig.virtualReferrer).toBeNull();
-  });
+      // @ts-expect-error testing partial data to ensure behaviour is as expected
+      const fixtureData: GetConfigProps = {
+        isAmp: false,
+        platform: 'canonical',
+        pageType: FRONT_PAGE,
+        brandName: 'BBC-News',
+        chartbeatDomain: 'bbc.co.uk',
+        env: 'test',
+        service: 'news',
+        origin: 'test.bbc.com',
+      };
 
-  it('should return null for virtualReferrer when isOnClient is false', () => {
-    isOnClient = false;
-
-    // @ts-expect-error testing partial data to ensure behaviour is as expected
-    const fixtureData: GetConfigProps = {
-      isAmp: false,
-      platform: 'canonical',
-      pageType: FRONT_PAGE,
-      data: {},
-      brandName: 'BBC-News',
-      chartbeatDomain: 'bbc.co.uk',
-      env: 'test',
-      service: 'news',
-      origin: 'test.bbc.com',
-    };
-
-    const chartbeatConfig = getConfig(fixtureData);
-    expect(chartbeatConfig.virtualReferrer).toBeNull();
+      const chartbeatConfig = getConfig(fixtureData);
+      expect(chartbeatConfig.virtualReferrer).toBeNull();
```

Comment:
just wondering what a virtualReferrer is as I haven't come across it before

Reply:
I found this comment in our docs which kinda explains: https://github.com/bbc/simorgh/blob/c3ab61eff7c41c2468c7fb88a7fe16cde5add208/src/app/lib/analyticsUtils/index.js#L167-L171